paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -221,4 +228,14 @@ internal class CustomerSheetActivityTest {
             isGooglePayEnabled = isGooglePayEnabled,
         )
     }
+
+    private class CustomerSheetTestDataImpl(
+        override val scenario: InjectableActivityScenario<CustomerSheetActivity>,
+        override val activity: CustomerSheetActivity
+    ) : CustomerSheetTestData
+
+    interface CustomerSheetTestData {
```

Comment:
I needed the interface for something in the other implementation, but if you don't, you can just expose the data class directly.